handlers.go

```diff
@@ -49,6 +51,13 @@ func TodoShow(w http.ResponseWriter, r *http.Request) {
 
 }
 
+var (
+	created = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "huyaber_created",
```

Comment:
do we really need "huyaber_" here? the app is already called that and I assume the metrics will be namespaced with the app name, no?